src/routes/Login/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import config from "../../config";
+
+const scopes = [
+  "user-read-playback-state",
+  "playlist-read-private",
+  "user-read-email",
+  "user-read-recently-played",
+];
+
+const responseType = "token";
+const redirectUrl = "http://localhost:3000/";
+const authEndpoint = "https://accounts.spotify.com/authorize";
+
+const Login = () => {
+  const navigate = useNavigate();
+  useEffect(() => {
```

Comment:
This useEffect will run multiple times whenever the component will re-render..is this desired behaviour?